pkg/oc/cli/admin/diagnostics/diagnostics/cluster/network/run_pod.go

```diff
@@ -103,7 +104,7 @@ func (d *NetworkDiagnostic) AvailableParameters() []types.Parameter {
 		},
 		{
 			Name:        FlagNetworkDiagTestPodPort,
-			Description: "Serving port on the diagnostic test pod",
+			Description: "Default port for diagnostic test pod",
```

Comment:
Would this be more accurate: "Override the default diagnostic test pod port"?

Reply:
done